tower/src/builder/mod.rs

```diff
@@ -134,6 +134,33 @@ impl<L> ServiceBuilder<L> {
         }
     }
 
+    /// Optionally add a new layer `T` into the [`ServiceBuilder`].
+    ///
+    /// ```
+    /// # use std::time::Duration;
+    /// # use tower::Service;
+    /// # use tower::builder::ServiceBuilder;
+    /// # #[cfg(feature = "timeout")]
+    /// # use tower::timeout::TimeoutLayer;
+    /// # #[cfg(all(feature = "timeout", feature = "util"))]
```

Comment:
as i mentioned on the free function's docs, the `cfg` attributes don't actually work:
```suggestion
    /// # use tower::timeout::TimeoutLayer;
```